aspects/aspects.go

```diff
@@ -171,38 +171,62 @@ func NewAspectBundle(account string, bundleName string, aspects map[string]inter
 	return aspectBundle, nil
 }
 
-func newAspect(bundle *Bundle, name string, aspectPatterns []map[string]string) (*Aspect, error) {
+func newAspect(bundle *Bundle, name string, aspectPatterns []interface{}) (*Aspect, error) {
 	aspect := &Aspect{
 		Name:           name,
 		accessPatterns: make([]*accessPattern, 0, len(aspectPatterns)),
 		bundle:         bundle,
 	}
 
 	readRequests := make(map[string]bool)
-	for _, aspectPattern := range aspectPatterns {
-		request, ok := aspectPattern["request"]
-		if !ok || request == "" {
+	for _, aspectPatternRaw := range aspectPatterns {
+		aspectPattern, ok := aspectPatternRaw.(map[string]interface{})
+		if !ok {
+			return nil, errors.New("each access pattern should be a map")
```

Comment:
we call these mostly patterns but also rules?  not really related to the PR but we should probably figure out if we use one or the other, it might relevant when we need to add syntax for filters

Reply:
At least in the specification we only use rule, never pattern. I don't think we ever "officially" endorsed one over the other before